glowbase/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04
+
+MAINTAINER Andrea Grandi <andrea@thisisglow.com>
+
+RUN apt-get update -y && \
+apt-get install -y python-dev build-essential python-pip libcurl4-gnutls-dev python2.7-dev libpq-dev supervisor nfs-common git nginx libjpeg-dev npm nodejs-legacy python-dev build-essential python-pip python2.7-dev wget unzip openjdk-6-jre git ruby nginx && \
```

Comment:
These are the list from the backend and frontend combined. We don't need to worry about size because the image will only be downlaoded by each host once.